solidjs-tailwind/src/pages/Home.jsx

```diff
@@ -16,10 +19,16 @@ const Home = () => {
     }
   });
 
+  createEffect(() => {
+    if (gists() && !gists.loading) {
+      setUserGists(gists());
+    }
+  });
+
   return (
     <div class="w-full min-h-[calc(100vh-70px)] flex flex-col-reverse lg:flex-row bg-gray-100">
       <aside class="w-full lg:w-96 bg-white p-8">
-        <UserGists gists={[]} />
+        <UserGists gists={userGists} />
```

Comment:
```suggestion
        <UserGists gists={userGists()} />
```

Works for me when I call userGists as a function